saleor/graphql/order/mutations/orders.py

```diff
@@ -483,3 +483,35 @@ def perform_mutation(cls, _root, info, amount, **data):
         if transaction.kind == TransactionKind.REFUND:
             order_refunded(order, info.context.user, amount, payment)
         return OrderRefund(order=order)
+
+
+class OrderSettingsUpdateInput(graphene.InputObjectType):
+    automatically_confirm_all_new_orders = graphene.Boolean(
+        required=True,
+        description="When enabled, all new orders from checkout "
+        "will be marked as unconfirmed.",
```

Comment:
"will be marked as unconfirmed" -> is this correct? This setting is about making them confirmed. I would also say what will happen if the setting is disabled.